docs/lambda/README.md

```diff
@@ -0,0 +1,171 @@
+# Object Lambda
+
+MinIO's Object Lambda implementation allows for transforming your data to serve unique data format requirements for each applications. For example, a dataset created by an ecommerce application might include personally identifiable information (PII). When the same data is processed for analytics, PII should be redacted. However, if the same dataset is used for a marketing campaign, you might need to enrich the data with additional details, such as information from the customer loyalty database.
+
+MinIO's Object Lambda, enables application developers to process data retrieved from MinIO before returning it to an application. You can register a Lambda Function target on MinIO, once successfully registered it can be used to transform the data for application GET requests on demand.
+
+This document focuses on showing a working example on how to use Object Lambda with MinIO, you must have [MinIO deployed in your environment](https://min.io/docs/minio/linux/operations/installation.html) before you can start using external lambda functions. You also must install Python version 3.8 or later for the lambda handlers to work.
+
```

Comment:
Isn't it possible to attach an Authorization header?

If so, adding that to the documentation would be great.

Reply:
yes that is possible @klauspost